### PR TITLE
Initial board implementation

### PR DESCRIPTION
**Pull request by alena-bartosh**

### CODE REVIEW

**Comment by Serg-Bartosh** [comment]
Well done.

File: src/board.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from src.cells import CellsType
```
Comment (s373r) [inline src/board.py]: import just from `cells`

**Review by Serg-Bartosh — DISMISSED** [review]

File: src/cells.py [hunk 2]
```diff
@@ -0,0 +1 @@
+CellsType = list[list[bool]]
```
Comment (s373r) [inline src/cells.py]: rename file to `cells_type`

File: src/point.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from dataclasses import dataclass
+from typing import Tuple
+
+PointType = Tuple[int, int]
```
Comment (s373r) [inline src/point.py]: we don't use `PointType` anymore -- please remove

**Review by s373r — CHANGES_REQUESTED** [review]

File: src/board.py (hunk 1 above)
Comment (alena-bartosh) [reply src/board.py]: done

File: src/cells.py (hunk 2 above)
Comment (alena-bartosh) [reply src/cells.py]: done

File: src/point.py (hunk 3 above)
Comment (alena-bartosh) [reply src/point.py]: Why? We will use it in the next PR

Comment (alena-bartosh) [reply src/point.py]: Done

Comment (s373r) [reply src/point.py]: fixed

**Review by s373r — APPROVED** [review]

**Review by Serg-Bartosh — APPROVED** [review]

**Review by Serg-Bartosh — APPROVED** [review]

**Pull request merged**